### ISSUE TITLE
cannot build on mingw32

### ISSUE THREAD

**Issue by novemberisms** [wren-lang/wren#595]
The build fails
Here's the output
```
       str src/vm/wren_core.wren
       str src/optional/wren_opt_random.wren
       str src/optional/wren_opt_meta.wren
       GCC src/optional/wren_opt_random.c  -O3 -std=c99
       GCC src/optional/wren_opt_meta.c    -O3 -std=c99
       GCC src/vm/wren_primitive.c         -O3 -std=c99
       GCC src/vm/wren_vm.c                -O3 -std=c99
       GCC src/vm/wren_debug.c             -O3 -std=c99
       GCC src/vm/wren_value.c             -O3 -std=c99
       GCC src/vm/wren_core.c              -O3 -std=c99
       GCC src/vm/wren_compiler.c          -O3 -std=c99
       GCC src/vm/wren_utils.c             -O3 -std=c99
       GCC lib/libwren.so                  -O3 -std=c99 -Wl,-soname,libwren.so
        ar lib/libwren.a                  rcu
       str src/module/repl.wren
       str src/module/timer.wren
       str src/module/os.wren
       str src/module/scheduler.wren
       str src/module/io.wren
       run util/build_libuv.py
       GCC src/cli/main.c                  -O3 -std=c99
       GCC src/cli/path.c                  -O3 -std=c99
src/cli/path.c: In function 'absolutePrefixLength':
src/cli/path.c:78:25: error: request for member 'chars' in something not a structure or union
     if (isSeparator(path->chars[2]))
                         ^~
util/wren.mk:202: recipe for target 'build/release/cli/path.o' failed
mingw32-make[1]: *** [build/release/cli/path.o] Error 1
mingw32-make[1]: Leaving directory 'C:/Users/Biao/Documents/wren/wren'
Makefile:16: recipe for target 'release' failed
mingw32-make: *** [release] Error 2
```

**Comment by bjorn** [comment]
Duplicate of #586 and fixed by #587.

**Comment by bjorn** [comment]
@munificent Time to close this duplicate. :-)
